src/platform/device.gni

```diff
@@ -119,6 +116,13 @@ declare_args() {
   }
 }
 
+declare_args() {
+  # Include wifi-paf to commission the device or not
+  # This is a feature of Wi-Fi spec that it can be enabled if wifi is enabled
+  # and the supplicant can support.
+  chip_device_config_enable_wifipaf = chip_enable_wifi && false
```

Comment:
&& false means false.

Maybe you can assert that if one is true the other one is as well instead.

Reply:
This is to be asked to disable it for now. The correct one should be:
	chip_device_config_enable_wifipaf = chip_enable_wifi && (chip_device_platform == "linux")

PAF should be enabled by default if:
	- Wi-Fi is enabled && 
	- It's in Linux (Since only wpa_supplicant has this feature for now)